components/SelectWithButtons.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="select-buttons">
+    <p class="select-buttons__title" v-if="title.length > 0">{{ title }}</p>
+    <v-btn-toggle
+      :value="value"
+      @change="$emit('change', $event)"
+      :color="color"
+      mandatory
+      borderless
+    >
+      <v-btn v-for="option in options" :key="option.value" :value="option.value">
```

Comment:
I think it would make more sense to have a slot here, instead of defining the options in the Javascript.  

Then we could call it with the following code:

```
<SelectWithButtons title="Select type of payments" :value="planTerm" @change="setTerm($event)">
  <v-btn value="month">Monthly</v-btn>
  <v-btn value="year">Yearly (save additional {{this.calculateSavings('year', 1)}})</v-btn>
</SelectWithButtons>
```